server/routes/api/admin.js

```diff
@@ -59,9 +59,14 @@ module.exports = (app) => {
             newUser.usn = usn;
             newUser.password = newUser.generateHash(password);
             if (role && role != "admin") {
-                // TODO: in the else part, throw an error "Cannot assign role 'admin' "
                 newUser.role = role;
             }
+            else if(role=='admin') {
+                return res.status(403).send({
```

Comment:
I think you understood it wrong. The if statement is so that an admin cannot create more admins.

Reply:
Yeah my bad, I meant to say that only.

Reply:
Leave it for now. I'll do it.

Reply:
Ok you change that later if necessary. I don't see why